### PR TITLE
default v0.5 version to v0.5-branch

### PR DESCRIPTION
**Pull request by gabrielwen**
Relates to #3159 

/assign @lluunn @kunmingg

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/3170)

### CODE REVIEW

File: bootstrap/pkg/apis/apps/group.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ import (
 const (
 	DefaultNamespace = "kubeflow"
 	// TODO: find the latest tag dynamically
-	DefaultVersion     = "master"
+	DefaultVersion     = "v0.5-branch"
```
Comment (lluunn) [inline bootstrap/pkg/apis/apps/group.go]: should we use branch name? or v0.5.0 tag?

Comment (lluunn) [reply bootstrap/pkg/apis/apps/group.go]: I think "v0.5.0"

Comment (gabrielwen) [reply bootstrap/pkg/apis/apps/group.go]: updated.

Comment (kunmingg) [reply bootstrap/pkg/apis/apps/group.go]: If we are about to create new release tag v0.5.1, should we point to v0.5.1 instead?
Seems there are several cherry picks on v0.5 branch post v0.5.0.

Comment (gabrielwen) [reply bootstrap/pkg/apis/apps/group.go]: sure, let me know.

**Pull request merged**